app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimStatusCards.kt

```diff
@@ -0,0 +1,65 @@
+package com.hedvig.app.feature.home.ui.claimstatus.composables
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.google.accompanist.pager.HorizontalPagerIndicator
+import com.google.accompanist.pager.rememberPagerState
+import com.hedvig.android.owldroid.graphql.HomeQuery
+import com.hedvig.app.feature.home.ui.claimstatus.data.ClaimStatusData
+
+@Composable
+fun ClaimStatusCards(homeQueryClaimStatusList: List<HomeQuery.ClaimStatus>) {
+    require(homeQueryClaimStatusList.isNotEmpty())
+
+    val claimStatusDataList: List<ClaimStatusData> = homeQueryClaimStatusList.map {
+        ClaimStatusData.fromHomeQueryClaimStatus(it)
+    }
+
+    ClaimStatusCards(claimStatusDataList)
+}
+
+@JvmName("ClaimStatusCardsWithConvertedInputData")
```

Comment:
Apparently having the same function with the same name and the only parameter being a list counts as the same JVM signature since I assume the generic type gets erased and it can't tell the difference.
This @JvmName setting manually feels a bit weird, is it ok to keep it like this?

Reply:
I think we can do the data remapping at a higher level, that will circumvent this issue entirely 😄 The type does indeed get erased, from the JVM's perspective it's only `List<Any>`